tests/phpunit/CRM/Member/Form/MembershipTest.php

```diff
@@ -1339,8 +1339,18 @@ public function testLineItemAmountOnSaleTax() {
     // ensure that the line-item values got unaffected
     $lineItem = $this->callAPISuccessGetSingle('LineItem', array('entity_id' => $membership['id'], 'entity_table' => 'civicrm_membership'));
     $this->assertEquals(1, $lineItem['qty']);
+    $this->assertEquals(50.00, $lineItem['unit_price']);
+    $this->assertEquals(50.00, $lineItem['line_total']);
     $this->assertEquals(5.00, $lineItem['tax_amount']); // ensure that tax amount is not changed
 
```

Comment:
Also check if `$lineItem['unit_price'] and $lineItem['line_total']` have not changed on Resubmitting the Edit Contribution form.